bin/i18n/sync-all.rb

```diff
@@ -98,11 +98,12 @@ def sync_out
   end
```

Comment:
Should we switch to this?
```suggestion
  def sync_out
    I18n::SyncOut.perform(options.slice(:testing))
  end
```